packages/store/src/patterns/patternMatchers.js

```diff
@@ -976,12 +1034,8 @@ const makePatternKit = () => {
       harden(specR);
       harden(newBase);
       return (
-        checkMatches(specB, newBase, check) &&
-        (rest === undefined ||
-          check(
-            matches(specR, rest),
-            X`Remainder ${specR} - Must match ${rest}`,
-          ))
+        checkMatches(specB, newBase, check, 'partial-base') &&
+        (rest === undefined || checkMatches(specR, rest, check, 'partial-rest'))
```

Comment:
Is there something we could expand *partial-base* and *partial-rest* to that would be more meaningful in context? I'm not precisely sure what the semantics is, so I don't have a suggestion, but these don't tell me much when looking at the relevant errors.

Reply:
Agreed. And that was my second try on terminology. Maybe brainstorm sometime soon?

Reply:
changed to required-parts optional-parts rest-parts. Much better!